mandatory/3-function-output.js

```diff
@@ -1,16 +1,19 @@
 // Add comments to explain what this function does. You're meant to use Google!
+/* The math.random() function return a random number between 0 and 1 but when we multiply by 100 it return a random number between 0 and 100;*/
+
```

Comment:
Great! Commenting is probably the most boring part of the job but it's necessary!